pkg/certificates/config_test.go

```diff
@@ -0,0 +1,433 @@
+package certificates
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+func contains(s, substr string) bool {
+	return strings.Contains(s, substr)
+}
+
+// TestParseConfigFileNotFound tests the ParseConfig function with a non-existent file.
+func TestParseConfigFileNotFound(t *testing.T) {
+	_, err := ParseConfig("non-existent-file.yaml")
+	if err == nil {
+		t.Error("expected error for non-existent file but got none")
+	}
+	if !strings.Contains(err.Error(), "reading config file") {
```

Comment:
Can we skip checking for an actual error message? Since we mainly want to test if the file is there or not.